chsdi/templates/htmlpopup/hug.mako

```diff
@@ -0,0 +1,124 @@
+<%inherit file="base.mako"/>
+
+<%def name="table_body(c, lang)">
+
+    <tr>
+      <td class="cell-left">${_('ch.bafu.hydrologie-untersuchungsgebiete.name')}</td>
+      <td>${c['attributes']['name'] or '-'}</td>
+    </tr>
+    <tr>
+      <td class="cell-left">${_('ch.bafu.hydrologie-untersuchungsgebiete.regimetyp')}</td>
+      <td>${c['attributes']['regimtyp'] or '-'}</td>
+    </tr>
+    <tr>
+      <td class="cell-left">${_('ch.bafu.hydrologie-untersuchungsgebiete.df')}</td>
+      <td>${c['attributes']['df'] or '-'}</td>
+    </tr>
+    <tr>
+      <td class="cell-left">${_('ch.bafu.hydrologie-untersuchungsgebiete.sc')}</td>
+      <td>${c['attributes']['sc'] or '-'}</td>
+    </tr>
+    <tr>
+      <td class="cell-left">${_('ch.bafu.hydrologie-untersuchungsgebiete.shape_area')}</td>
+      <td>${c['attributes']['shape_area']}</td>
+    </tr>
+    <tr>
+      <td class="cell-left">${_('ch.bafu.hydrologie-untersuchungsgebiete.url')}</td>
```

Comment:
No need to check for existance? (sorry is this is clear to you. No need to answer if it is).